providers/openshift/provider.go

```diff
@@ -140,33 +140,35 @@ func (p *OpenShiftProvider) setCA(paths []string) error {
 	return nil
 }
 
+// encodeSARWithScope adds a "scopes" array to the SAR if it does not have one already, and outputs
+// as pretty encoding.
+func encodeSARWithScope(json *simplejson.Json) ([]byte, error) {
+	if len(json.Get("scopes").MustArray()) == 0 {
+		json.Set("scopes", []interface{}{})
+	}
+	return json.Encode()
+}
+
 // parseSubjectAccessReviews parses a list of SAR records and ensures they are properly scoped.
 func parseSubjectAccessReviews(review string) ([]string, error) {
 	if len(review) == 0 {
 		return nil, nil
 	}
+
+	review = strings.TrimSpace(review)
```

Comment:
I would do this before the length check.

Reply:
Hm I suppose it depends on if you think `--openshift-sar=''` vs ``--openshift-sar='    '`` vs ` ` (nothing passed) are equivalent.  I do not have a strong opinion.